src/sentry/auth/helper.py

```diff
@@ -628,6 +628,7 @@ def _handle_existing_identity(self, auth_identity, identity):
             return HttpResponseRedirect(auth.get_login_redirect(self.request))
 
         self.clear_session()
+        metrics.incr('login-sso.success', tags={'provider': self.provider.provider})
```

Comment:
We could also add this up in `_handle_unknown_identity` where they are authenticated after linking their account, or potentially track this separately. But right now that's not tracked